SokolTextGame/Commands.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SokolTextGame
+{
+    internal interface ICommand
+    {
+        void Execute(string[] words, World world);                
+    }
+    public class Echo : ICommand
+    {
+        void ICommand.Execute(string[] words, World world)
+        {
+            string text = string.Join(" ", words.Skip(1));
+            Console.WriteLine(text);
+        }
+    }
+
+    public class Exit : ICommand
+    {
+        void ICommand.Execute(string[] words, World world)
```

Comment:
Please remove ICommand. from the Execute signature and make the method public instead.